_data/frameworks.yml

```diff
@@ -66,7 +66,7 @@
       angular2: <div [textContent]="message">
       angular1: <div ng-bind="message">
       ember: <div textContent="{{ message }}">
-      polymer: <div textContent$="{{ message }}">
+      polymer: <div text-content="{{ message }}">
```

Comment:
Why not just: `<div>{{ message }}</div>` See https://www.polymer-project.org/1.0/docs/devguide/data-binding#binding-to-text-content